src/Ng/packages.config

```diff
@@ -11,7 +11,7 @@
   <package id="Microsoft.Web.Xdt" version="2.1.1" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
-  <package id="NuGet.Core" version="2.8.5" targetFramework="net45" />
+  <package id="NuGet.Core" version="2.10.1" targetFramework="net45" />
```

Comment:
It seems like there is a general dislike of NuGet.Core -- is there a newer alternative I should be using for parsing target frameworks? Perhaps I can have a seperate PR as switching away from NuGet.Core like [Maarten's PR in the gallery](https://github.com/NuGet/NuGetGallery/pull/2783)?